umbral-pre/src/hashing_ds.rs

```diff
@@ -45,9 +49,53 @@ pub(crate) fn hash_capsule_points(
         .finalize()
 }
 
-pub(crate) fn hash_to_cfrag_verification(points: &[CurvePoint]) -> NonZeroCurveScalar {
+/// Calculates the challenge scalar for the proof of reencryption.
+///
+/// Note: this function is only public for documenting purposes
+/// (see [`ReencryptionEvidence`]).
```

Comment:
So instead of exposing all private items with `cargo doc --document-private-items`, we're changing their visibility on case-by-case basis, correct?

Reply:
Yes, I don't want to uncover the whole internal structure.

But now that I think about it, maybe I should just remove the note. Publishing, and committing to, a specific way of calculating the challenge value is really no different than publishing the function that does it.